cypress/pageobject/Login/LoginPage.ts

```diff
@@ -88,6 +111,68 @@ class LoginPage {
   verifyForgotPasswordHeading(text: string[]): void {
     cy.verifyContentPresence("#forgot-password-heading", text);
   }
+
+  clickContributeOnGitHub(): void {
+    cy.get(this.contributeOnGitHubLink).scrollIntoView().click();
+  }
+
+  clickThirdPartyLicense(): void {
+    cy.get(this.thirdPartyLicenseLink).scrollIntoView().click();
+  }
+
+  switchLanguage(languageCode: string): void {
+    cy.get(this.languageSelector).select(languageCode);
+  }
+
+  getSubmitButton() {
+    return cy.get(this.submitButtonSelector);
+  }
+
+  getSidebarItems() {
+    return cy.get(`${this.sidebarSelectors.care}, ${this.sidebarSelectors.goal}, ${this.sidebarSelectors.footer_body}`);
+  }
+
+  switchLanguageAndVerifyButtonText(languageMappings: LanguageMapping): void {
+    Object.entries(languageMappings).forEach(([languageCode, texts]) => {
+      cy.get(this.languageSelector)
```

Comment:
The logic below suggests you are trying to use language selector on profile page, but here it seems like you are referring to login page 😅